pkg/resource/vsc/scan.go

```diff
@@ -38,7 +37,7 @@ func (s *Scan) EDP() (resource.EDPMeasurement, error) {
 
 	for _, vsc := range s.vscs.Items {
 		if vsc.Status == nil {
-			errs = append(errs, fmt.Errorf("%w: %s", ErrStatusNotSet, vsc.Name))
+			// Skip VSCs without status
```

Comment:
Should we maybe log it?

Reply:
I was also thinking about it, but imho we should address logging in a different issue. Currently there is no logging at all in any of the scanner/scan/collector implementations